meeting/meetingPane.js

```diff
@@ -805,6 +805,8 @@ module.exports = {
         iframe.setAttribute('src', target.uri)
         // iframe.setAttribute('style', 'height: 350px; border: 0; margin: 0; padding: 0; resize:both; width: 100%;')
```

Comment:
While we are in this section of the file might as well remove the old commented out style. Not particularly necessary but calling out the (boy scout rule)[https://medium.com/@biratkirat/step-8-the-boy-scout-rule-robert-c-martin-uncle-bob-9ac839778385]